src/main/java/com/github/thedeathlycow/frostiful/mixins/compat/overflowingbars/present/FreezingHearts.java

```diff
@@ -0,0 +1,26 @@
+package com.github.thedeathlycow.frostiful.mixins.compat.overflowingbars.present;
+
+import net.fabricmc.loader.api.FabricLoader;
+import net.minecraft.client.gui.hud.InGameHud;
+import net.minecraft.entity.player.PlayerEntity;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+@Mixin(InGameHud.HeartType.class)
+public abstract class FreezingHearts {
+
+    @Inject(
+            method = "fromPlayerState",
+            at = @At("TAIL"),
+            cancellable = true
+    )
+    private static void setFreezingHeartsWhenFrozen(PlayerEntity player, CallbackInfoReturnable<InGameHud.HeartType> cir) {
+        if (FabricLoader.getInstance().isModLoaded("overflowingbars")) return;
```

Comment:
I think the logic on this line is inverted from what it should be, since it is in the `present` package for overflowing bars. I think this is what is causing the freezing hearts to not be rendered when at -100% temp scale. Did you mean to put a check for colourful hearts here maybe?

Reply:
Oh ups, that's an issue for sure, that was me mixing around the files when adding the condition on colourful hearts, to avoid duplicating mixin application since overflow bars' display takes over colourful hearts'